drivers/serial/uart_stm32.c

```diff
@@ -674,6 +675,11 @@ static void uart_stm32_poll_out_visitor(const struct device *dev, void *out, pol
 		/* Enable TC interrupt so we can release suspend
```

Comment:
Comment is no longer on the appropriate code piece

Reply:
Fixed, thanks